charts/jenkins-kubernetes-agents/Chart.yaml

```
@@ -3,5 +3,5 @@ apiVersion: v2
 name: jenkins-kubernetes-agents
 description: A Helm chart for using Jenkins Kubernetes agents on a remote cluster
 type: application
-version: 0.5.1
+version: 0.5.2
```

```suggestion
version: 1.0.0
```

This change is not backward compatible so it is a major bump (it means "if applied then the current behavior changes")